### ISSUE TITLE
Manually installing forge       [feature]

### ISSUE THREAD

**Issue by devomaa** [MultiMC/MultiMC5#2685]
_This is not an issue, this is an suggestion/feature request_

System Information: Linux

KDE Plasma Version: 5.15.4
KDE Frameworks Version: 5.56.0
Qt Version: 5.12.2
Kernel Version: 5.0.0-16-generic
OS Type: 64-bit
Processors: 4 × Intel® Core™ i5-4210U CPU @ 1.70GHz
Memory: 7,7 GiB

-----------------------------
MultiMC version: 0.6.5-develop-1227

Operating System: Kubuntu 19.04

Summary of the issue or suggestion: Make manually forge installing happen
----------------------------------------------

What should happen:
------------------------------

Basically add an "Manual Install" tab in "Edit Instance" option of an instance

Additional Info:
---------------------------
nope

**Comment by jikuja** [comment]
May I ask what is your use case for this?

Forge for 1.13.x or 1.14.x MC?

**Comment by peterix** [comment]
There is no way to 'manually install' by 'automating' things. That just makes no sense and there's no reason for doing it.

**Comment by peterix** [comment]
I'd be OK with investigating something that has more details than this. Basically, I don't see why this would be a good idea, and what it would accomplish.

You can manually do all kinds of things and you don't need a button for it.
